rosdep/python.yaml

```diff
@@ -1823,7 +1823,9 @@ python-oauth2:
   debian: [python-oauth2]
   fedora: [python-oauth2]
   gentoo: [dev-python/oauth2]
-  ubuntu: [python-oauth2]
+  ubuntu:
+    pip:
+      packages: [oauth2]
```

Comment:
This kind of change is not feasible since there might be packages released into e.g. Indigo using this key. With the switch to `pip` those would not be releasable anymore. Please change the patch to enumerate the Ubuntu distros which contain the Debian package. For newer ROS distros maybe there is a package with a different name?